app/controllers/api/v1/reviews_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Api::V1::ReviewsController < ApiController
+  def index
+    product = Product.find(params[:product_id])
+    reviews = product.reviews
+    reviewwithuser = []
+    reviews.each do |review| 
+      newreview = {rating: review.rating,
+      body: review.body,
+      username: User.find(review.user_id)
```

Comment:
the key here is username, but its not actually a username, its a user object. Update this key so that it makes sense later on in your react code